content/en/docs/setup/install/multicluster/primary-remote/index.md

```diff
@@ -156,6 +214,25 @@ spec:
 EOF
 {{< /text >}}
 
+Apply the configuration to `cluster2`:
+
+{{< text bash >}}
+$ istioctl install --context="${CTX_CLUSTER2}" -f cluster2.yaml
+{{< /text >}}
+
+{{< /tab >}}
+{{< tab name="Helm" category-value="helm" >}}
+
+Install the `istiod-remote` chart in `cluster2` using standard `helm` commands:
+
+{{< text bash >}}
+helm install istiod-remote istio/istiod-remote --set global.multiCluster.clusterName=cluster2 --set global.network=network1 --set istiodRemote.injectionPath=/inject/cluster/cluster2/net/network1 --set global.externalIstiod=true --set global.configCluster=true --set global.remotePilotAddress=${DISCOVERY_ADDRESS} --set pilot.enabled=false -n istio-system --kube-context ${CTX_CLUSTER2}
```

Comment:
@bleggett @howardjohn one of the improvements we could potentially make would be to consolidate some of these vars in the remote chart - ex: configCluster and pilot.enabled maybe?

Reply:
Should we still publish these steps for now and update the doc as a follow-up later, or wait until the `istiod-remote` chart has been updated with the new values before publishing in the first place? Not sure if that will make it to `1.23`.

Reply:
I'd be inclined to document actual/current state and fix it later.